bitwarden-to-keepass.py

```diff
@@ -64,7 +64,7 @@ def bitwarden_to_keepass(args):
                 except Exception as e:
                     if 'already exists' in str(e):
                         is_duplicate_title = True
-                        continue
+                        break
```

Comment:
Weird that you ended up in infinite loop. It should not happen - well, at least not due to having multiple items with the same name. `continue` here saves the same item with bitwardens uuid appended to the name. 
https://github.com/davidnemec/bitwarden-to-keepass/blob/9c27ba943c52558622b6108bb84377dbce46adb8/bitwarden-to-keepass.py#L54

Can you maybe check what is causing this?

Anyways, using `break` here will result in missing items in KeePass export.

Reply:
You're right, `break` does indeed skip some items in the KeePass export, I only noticed that now.
I'll try to find out what is causing `continue` to come back to the same `'already exists'` error. The script ran fine before I touched it so it must be something on my end.

Reply:
I didn't manage to replicate the infinite loop. My experiments only confirmed that this logic does indeed work as intended.
I decided to revert this commit.